syne_tune/blackbox_repository/conversion_scripts/scripts/hpob_import.py

```diff
@@ -0,0 +1,606 @@
+            if dataset in result_dict[search_space].keys():
+                result_dict[search_space][dataset]["X"] += test_dict[search_space][
+                    dataset
+                ]["X"]
+                result_dict[search_space][dataset]["y"] += test_dict[search_space][
+                    dataset
+                ]["y"]
+            else:
+                result_dict[search_space][dataset] = test_dict[search_space][dataset]
+
+    return result_dict
+
+
+class HPOBRecipe(BlackboxRecipe):
+    def __init__(self, search_space, name):
+        super(HPOBRecipe, self).__init__(
+            name=name,
+            cite_reference="HPO-B: A Large-Scale Reproducible Benchmark for Black-Box HPO based on OpenML."
+            " Sebastian Pineda-Arango and Hadi S. Jomaa and Martin Wistuba and Josif Grabocka, 2021.",
+        )
+        self.search_space = search_space
+
+    def _generate_on_disk(self):
+        generate_hpob(self.search_space)
+
+
+class HPOBRecipe4796(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_4796, "hpob_4796")
+
+
+class HPOBRecipe5527(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5527, "hpob_5527")
+
+
+class HPOBRecipe5636(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5636, "hpob_5636")
+
+
+class HPOBRecipe5859(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5859, "hpob_5859")
+
+
+class HPOBRecipe5860(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5860, "hpob_5860")
+
+
+class HPOBRecipe5891(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5891, "hpob_5891")
+
+
+class HPOBRecipe5906(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5906, "hpob_5906")
+
+
+class HPOBRecipe5965(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5965, "hpob_5965")
+
+
+class HPOBRecipe5970(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5970, "hpob_5970")
+
+
+class HPOBRecipe5971(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5971, "hpob_5971")
+
+
+class HPOBRecipe6766(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_6766, "hpob_6766")
+
+
+class HPOBRecipe6767(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_6767, "hpob_6767")
+
+
+class HPOBRecipe6794(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_6794, "hpob_6794")
+
+
+class HPOBRecipe7607(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_7607, "hpob_7607")
+
+
+class HPOBRecipe7609(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_7609, "hpob_7609")
+
+
+class HPOBRecipe5889(HPOBRecipe):
+    def __init__(self):
+        super().__init__(SEARCH_SPACE_5889, "hpob_5889")
+
+
+if __name__ == "__main__":
+    HPOBRecipe4796_instance = HPOBRecipe4796()
```

Comment:
Can we just create a list `recipes = [HPOBRecipe4796, HPOBRecipe5527]` and than loop over the list 
`for recipe in recipes:
    instance = recipe()
    instance.generate(upload_on_hub=false)
`

Makes the code more readable